cypress/support/commands.js

```diff
@@ -32,10 +32,56 @@ Cypress.Commands.add('getByDataCy', (selector) => {
   cy.get(`[data-cy="${selector}"]`);
 });
 
-Cypress.Commands.add('register', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
-  cy.request('POST', '/users', {
-    email,
-    username,
-    password
+Cypress.Commands.add('register',
+  (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
+    cy.request('POST', '/users', {
+      email,
+      username,
+      password
+    });
+  });
+
+const imgUrl = 'https://static.productionready.io/images/smiley-cyrus.jpg';
+
+Cypress.Commands.add('login', (email, username, password) => {
+  cy.request('POST', '/api/users', {
+    user: {
+      email,
+      username,
+      password
+    }
+  }).then((response) => {
+    const user = {
+      bio: response.body.user.bio,
+      effectiveImage: imgUrl,
+      email: response.body.user.email,
+      image: response.body.user.image,
+      token: response.body.user.token,
+      username: response.body.user.username
+    };
+    window.localStorage.setItem('user', JSON.stringify(user));
+    cy.setCookie('auth', response.body.user.token);
```

Comment:
Ensure that setting the 'auth' cookie with the user's token is necessary for your application's authentication flow. This step is crucial for maintaining the user's session.